src/screens/ImportMethod/ImportMethodContainer.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import React, { PureComponent } from 'react'
+import textConstants from '../../locales/en'
+import ImportMethod from './ImportMethod'
+import {mnemonic, private_key} from '../../images'
+
+class ImportMethodContainer extends PureComponent {
+  handleCreateWallet = () => {
+    const {navigate} = this.props.navigation
+    navigate('SetAccountPassword')
+  }
+
+  handleSelectImportMethod = ({ screen, title }) => () => {
+    const { navigate } = this.props.navigation
+    navigate(screen)
+  }
+
+  render() {
+    return (
```

Comment:
Do we really need render method in container? 
Maybe it is enough just to wrap ImportMethod as it was done in `src/screens/Login/Start/StartContainer.js`?

Also pressing on buttons in StoryBook should not fail app (red screen with error). All methods must have default values like `() => {}`